.readme-partials.yaml

```diff
@@ -0,0 +1,259 @@
+custom_content: |
+  #### Creating a topic
+
+  With Pub/Sub Lite you can create topics. A topic is a named resource to which messages are sent by
+  publishers. Add the following imports at the top of your file:
+
+  ```java
+  import com.google.cloud.pubsublite.*;
+  import com.google.cloud.pubsublite.proto.Topic;
+  import com.google.cloud.pubsublite.proto.Topic.*;
+  import com.google.protobuf.util.Durations;
+  ```
+  Then, to create the topic, use the following code:
+
+  ```java
+  CloudRegion region = CloudRegion.of("us-central1");
+  TopicPath topicPath =
+      TopicPaths.newBuilder()
+        .setZone(CloudZone.of(region, 'b'))
+        .setProjectNumber(ProjectNumber.of(123L)) // Your project number.
+        .setTopicName(TopicName.of("my-new-topic"))
+        .build();
+
+  Topic topic =
+      Topic.newBuilder()
+        .setPartitionConfig(
+          PartitionConfig.newBuilder()
+            .setScale(1) // Set publishing throughput to 1*4 MiB per sec. This must be 1-4.
+            .setCount(2)) // The number of partitions.
+        .setRetentionConfig(
+          RetentionConfig.newBuilder()
+            .setPeriod(Durations.fromDays(7))
+            .setPerPartitionBytes(100000000000L)) // 100 GiB. This must be 30 GiB-10 TiB.
```

Comment:
This should be 100L * 1024L * 1024L. Please reference the updated samples in 

https://github.com/googleapis/java-pubsublite/blob/caa96a86feea501daa2390a1a3ca92142b8f14b6/samples/snippets/src/main/java/pubsublite/CreateTopicExample.java#L83

Reply:
Sounds good. I just cut this content from the existing README. Will adjust accordingly.